sdk/src/Services/DynamoDBv2/Custom/DataModel/BaseOperationConfig.cs

```diff
@@ -0,0 +1,100 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+namespace Amazon.DynamoDBv2.DataModel
+{
+    /// <summary>
+    /// Base class for operation-specific configurations for DynamoDB object persistence operations.
+    /// </summary>
+    /// <remarks>
+    /// This should only contain members that are relevant to all object persistence operations, 
+    /// anything operation-specific should be added to derived classes.
+    /// </remarks>
+#if NET8_0_OR_GREATER
+    [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCode(Custom.Internal.InternalConstants.RequiresUnreferencedCodeMessage)]
+#endif
+    public abstract class BaseOperationConfig
+    {
+        /// <summary>
+        /// Indicates which DynamoDB table to use. This overrides the table specified 
+        /// by the <see cref="DynamoDBTableAttribute"/> on the .NET objects that you're saving or loading.
+        /// </summary>
+        public string OverrideTableName { get; set; }
+
+        /// <summary>
+        /// Directs <see cref="DynamoDBContext"/> to prefix all table names
+        /// with a specific string. If this is null or empty, no prefix is used 
+        /// and default table names are used.
+        /// </summary>
+        public string TableNamePrefix { get; set; }
+
+        /// <summary>
+        /// The object persistence model API relies on an internal cache of the DynamoDB table's metadata to construct and validate 
+        /// requests. This controls how the cache key is derived, which influences when the SDK will call
+        /// IAmazonDynamoDB.DescribeTable(string) internally to populate the cache.
```

Comment:
Can you also wrap `IAmazonDynamoDB.DescribeTable(string)` in a `cref`

Reply:
See https://github.com/aws/aws-sdk-net/pull/3380#discussion_r1672791987